### PR TITLE
Add `multiplatform-viewmodel` library

### PR DESCRIPTION
**Pull request by dcvz**
Adds the `multiplatform-viewmodel` library to the Architecture section

[Library Link](https://github.com/doublesymmetry/multiplatform-viewmodel)

`multiplatform-viewmodel` is a library providing a shared `ViewModel` to use in your presentation layer

### CODE REVIEW

**Comment by AAkira** [comment]
Sorry I'm late to see this.
Nice!

File: README.md [hunk 1]
```diff
@@ -882,6 +882,13 @@
 ![badge][badge-jvm]
 ![badge][badge-js]
 
+* [Premo](https://github.com/doublesymmetry/multiplatform-viewmodel) — Shared ViewModel in Kotlin Multiplatform
```
Comment (dmdevgo) [inline README.md:885]: Premo in the title is copy-paste?)

Comment (AAkira) [reply README.md:885]: @dmdevgo 
Thank you. I missed it and will correct it.

Comment (AAkira) [reply README.md:885]: ✅ 
https://github.com/AAkira/Kotlin-Multiplatform-Libraries/pull/108

**Pull request merged**